tests/cover/test_stateful.py

```diff
@@ -173,15 +173,45 @@ def __init__(self, value):
 class DepthMachine(RuleBasedStateMachine):
     charges = Bundle(u'charges')
 
-    @rule(targets=(charges,), child=charges)
-    @rule(targets=(charges,), child=none())
-    def charge(self, child):
-        return DepthCharge(child)
+    # double-rule is deprecated
+    with Settings(strict=False):
```

Comment:
I'd like to see a test that asserts this deprecation. There are a bunch of these in tests/cover/test_deprecation.py